src/main/java/seedu/address/ui/RightSideBar.java

```diff
@@ -5,23 +5,20 @@
 import static seedu.address.logic.parser.session.CliSyntax.PREFIX_PERIOD;
 
 import java.util.List;
-import java.util.logging.Logger;
 
 import javafx.fxml.FXML;
 import javafx.geometry.Pos;
 import javafx.scene.control.Label;
 import javafx.scene.control.ListCell;
 import javafx.scene.control.ListView;
 import javafx.scene.layout.AnchorPane;
-import seedu.address.commons.core.LogsCenter;
 import seedu.address.logic.Logic;
 import seedu.address.logic.commands.CommandResult;
 import seedu.address.model.schedule.Schedule;
 import seedu.address.model.session.Session;
 
 public class RightSideBar extends UiPart<AnchorPane> {
     private static final String FXML = "RightSideBar.fxml";
-    private final Logger logger = LogsCenter.getLogger(RightSideBar.class);
```

Comment:
No need to remove logger

Reply:
Is there a point in keeping it? Since it's not being used I think